internal/stores/gcs.go

```diff
@@ -63,9 +63,12 @@ func (s *GCSStore) GetOptions() Options {
 	return s.Options
 }
 
-// TODO(discentem): #34 largely copy-pasted from stores/local/local.go. Can be consolidated
-// Upload generates the metadata, writes it to disk and uploads the file to the GCS bucket
+// Upload generates the metadata, writes it s.fsys and uploads the file to the GCS bucket
 func (s *GCSStore) Upload(ctx context.Context, objects ...string) error {
+	if s.Options.MetaDataFileExtension == "" {
+		s.Options.MetaDataFileExtension = "cfile"
+	}
+
```

Comment:
This should be handled by cobra/viper in the form of a default. We shouldn't have to set a default for this in every storage provider.

Reply:
I ran into a bad behavior in the unit test where if `s.Options.MetaDataFileExtension` isn't set it's an empty string so `a.cfile` just becomes `a`. Maybe if `s.Options.MetadataFileExtension` is `""` this should be an error?